gammapy/visualization/utils.py

```diff
@@ -293,9 +297,32 @@ def plot_distribution(
         n, bins, _ = axe.hist(d, **kwargs_hist)
 
         if func is not None:
+            kwargs_plot_fit = {"label": "Fit"}
             centers = 0.5 * (bins[1:] + bins[:-1])
 
-            pars, cov, infodict, message, _ = curve_fit(func, centers, n, **kwargs_fit)
+            if func == "norm":
+
+                def func(x, mu, sigma):
+                    return norm.pdf(x, mu, sigma)
+
+                pars, cov, infodict, message, _ = curve_fit(
+                    func, centers, n, **kwargs_fit
+                )
+
+                mu, sig = pars[0], pars[1]
+                err_mu, err_sig = cov[0][0] ** 2, cov[1][1] ** 2
+
+                label_norm = (
+                    r"$\mu$ = {:.2f} ± {:.2E}, $\sigma$ = {:.2f} ± {:.2E}".format(
+                        mu, err_mu, sig, err_sig
+                    )
+                )
+                kwargs_plot_fit["label"] = label_norm
```

Comment:
Doesn't this clutter a bit the output plot?

Reply:
Because you said that I change the label so that it is written on 2 lines. It looks like this :
![plot_dist_norm_label](https://github.com/gammapy/gammapy/assets/108189156/05abb415-9229-4059-b1a7-870a66960686)

I suppose it's ok, right ?